content/development-guide/api/command.md

```diff
@@ -78,15 +73,14 @@ public static SampleJokeCommand : IChildCommand
 	
 	public void Execute (ICommandContext context)
 	{
-		context.User.SendMessage ("<insert generic joke here>", System.Drawing.Color.Yellow);
+		context.User.SendMessage ("<insert generic joke here>", Rocket.API.Drawing.Color.Yellow);
 	}
 }
 ```
 
-And then in your base command set ChildCommands to this
-
+Then set `ChildCommands` to this in your base command.
 ```csharp
-public IChildCommand [] ChildCommands { get; } = new IChildCommand []
+public IChildCommand [] ChildCommands { get; } =
```

Comment:
I dont think this will compile

Reply:
It's working fine with normal arrays and honestly I don't know if it's working on custom interfaces, or not.

It's just a compile time trick. `public string[] Bacı = { "B", "a", "c", "ı" };` and `public string[] Bacı = new string[] { "B", "a", "c", "ı" };` both will be compiled as `public string[] Bacı = new string[] { "B", "a", "c", "ı" };`. If it's not working on custom interfaces, forgive me :D

Reply:
It does not work on interfaces.

Reply:
Rest in bepis then :(